lib/OnyxUtils.ts

```diff
@@ -633,7 +631,7 @@ function keysChanged<TKey extends CollectionKeyBase>(
             // send the whole cached collection.
             if (isSubscribedToCollectionKey) {
                 if (subscriber.waitForCollectionCallback) {
-                    subscriber.callback(cachedCollection);
+                    subscriber.callback(cachedCollection, undefined);
```

Comment:
Not sure why, it was always like this before – however, I can make this change to pass the collection key

Reply:
True, we never passed the key to collection callbacks. But now that we unified the signature type of all callbacks, i think it makes sense to pass the collection key rather than just `undefined`.